bundle/manifests/limitador-operator-metrics_v1_service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
In the bundle dir, there are now two metric services, right? one should be removed.

Reply:
Oops yes good catch